test/rocblas_reference.hpp

```diff
@@ -120,6 +131,10 @@ namespace rocwmma
         {
             return rocblas_datatype_i8_r;
         }
+        constexpr static inline rocblas_computetype computeType()
```

Comment:
These are not behind the ROCBLAS_DATA_TYPE_INVALID guard.
Might be best to remove these functions returning rocblas_compute_type_invalid. 
The compiler will complain if the function is called and doesn't exist.  This should be enough message to the user that the compute type is not supported